OrbitGl/TimeGraph.cpp

```diff
@@ -708,6 +706,7 @@ std::shared_ptr<GpuTrack> TimeGraph::GetOrCreateGpuTrack(
   std::shared_ptr<GpuTrack> track = gpu_tracks_[timeline_hash];
   if (track == nullptr) {
     track = std::make_shared<GpuTrack>(this, string_manager_, timeline_hash);
+    track->SetTimeline(string_manager_->Get(timeline_hash).value_or(""));
```

Comment:
When looking at other usages of `track->SetName` and `track->setLabel` in the code base, it seems like we assign the raw value to the name and the processed human-readable value to the label. Here both get the processed value. So I'm wondering, do we actually need the distinction?

Reply:
Yes, you are right. I will use the "raw" name (which is the timeline name) for SetName and the descriptive name for SetLabel. I wasn't sure where that made a difference, but it's actually used when filtering tracks. So, the distinction is actually important. Thanks.

Reply:
Except that we don't seem to filter GPU tracks, but that's something we might want to change?

Reply:
Done for the purpose of this PR, I think.